conda/recipes/dask-cudf/meta.yaml

```diff
@@ -25,19 +25,16 @@ build:
 requirements:
   host:
     - python
+    - cudatoolkit {{ cuda_version }}.*
```

Comment:
If this is now cuda dependent, then the build string must also reflect that.

I'm not sure this PR is necessary though since I don't think dask-cudf depends on cuda directly.

Reply:
Yeah I'm trying to resolve the issue where we had 2 `dask-cuda` packages uploaded with the same name. My guess is these came from 2 jobs that were otherwise identical except for CUDA version. Ok with solving this another way (like altering the build string)

Reply:
Perhaps altering https://github.com/rapidsai/cudf/blob/branch-21.10/ci/cpu/prebuild.sh to only upload dask-cudf once  per Python (so only when CUDA=11.0 for example) is the best solution.